android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt

```diff
@@ -141,6 +142,10 @@ class XMTPModule : Module() {
             val client = Client().buildFromBundle(bundle = bundle, options = options)
             clients[client.address] = client
             client.address
+        } catch (e: Exception) {
+            Log.e("createFromKeyBundle", "Failed to create client: $e")
+            ""
```

Comment:
Similar comment to what I put here: https://github.com/xmtp/xmtp-react-native/pull/70#discussion_r1253540807

I actually think you want to keep the throw. If a client isn't created there will be lots of other issues. It's better to fail with a throw than to return an empty string.
```suggestion
            throw XMTPException("Failed to create client: $e")
```

Reply:
Make sense, since it replicates what we did on iOS. Good one. Changed already.